src/mongo/unittest/unittest.h

```diff
@@ -368,6 +411,42 @@ class Test {
     logger::MessageLogDomain::AppenderAutoPtr _captureAppender;
 };
 
+class Benchmark : public Test {
+protected:
+    int trial_num = 0;
+
+public:
+    /**
+     * Called to prepare the benchmark for execution
```

Comment:
Prefer to phase the first sentence of method comments like "Prepares the ..."

They should say what the function does, not why you would call it. If further explanation is needed (which should generally be rare),  do so after the first sentence.